src/widgetastic_patternfly5/components/ouia.py

```diff
@@ -0,0 +1,112 @@
+from widgetastic.ouia import OUIAGenericView
```

Comment:
`src/widgetastic_patternfly5/ouia.py`

Let's put ouia in a parent, not under component for consistency.

Reply:
@digitronik makes sense, moved